components/card.css

```diff
@@ -0,0 +1,231 @@
+.card{    
+    width: 35rem;
+    border-radius: 4px;
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
```

Comment:
You could add the box sizing border box as a universal selector
*, *:before, *:after {
  box-sizing: border-box;
}

Reply:
Okay, will place it in reset file.